src/state/models/api.ts

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable camelcase */
+
+// --- all services ---
+
+export interface ErrorDto {
+	// The time at which the error occurred.
+	readonly timestamp: string // 2006-01-02T15:04:05+07:00
+
+	// An internal trace id assigned to the error. Used to find logs associated with errors across our services. Display to the user as something to communicate to us with inquiries about the error.
+	readonly requestid: string // a8b7c6d5
+
+	// A keyed description of the error. We do not write human-readable text here because the user interface will be multi-language.
+	readonly message: string // attendee.owned.notfound or similar
+
+	// Optional additional details about the error. If available, will usually contain English language technobabble.
+	readonly details: object // map[string][]string
+}
+
+// --- attendee service ---
+
+export interface AttendeeDto {
+	readonly id: bigint | null
```

Comment:
The thing with these bigints is TS erases types during compilation, there is no reflection mechanism. So HTTP libraries will just use `JSON.parse` and JSON.parse isn't aware of bigint. So while you declare it here as bigint, the value will in fact be number.

Reply:
However, since you are already using ints of the smallest currency unit for currencies, plain numbers should suffice as they are reliable for integers up until Number.MAX_SAFE_INTEGER = `2^53 - 1` or `9,007,199,254,740,991` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER). So unless we are charging some quite astronomical fees they will easily fit inside a standard number. I'd change the bigints to number for simplicity.